packages/obonode/obojobo-chunks-text/components/line/__snapshots__/editor-component.test.js.snap

```diff
@@ -3,6 +3,7 @@
 exports[`Line Editor Node Line builds the expected component 1`] = `
 <span
   className="text align-0"
+  data-hanging-indent={0}
```

Comment:
This hanging-indent is a 0, but the others are either false or true.  Maybe check the values in the editor-component.test file?